### PR TITLE
Fix unselect. Add unit tests

### PR DESCRIPTION
**Pull request by DmitriyNoa**
- Fix Select component not resetting value when null is passed.
- Add unit tests

### CODE REVIEW

**Review by Nokel81 — DISMISSED** [review]

File: src/renderer/components/select/select.test.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    rerender(<Select value={options[1].value} onChange={onChange} options={options} />);
+
+    expect(container.querySelector(".Select__single-value").textContent).toBe(options[1].label);
+
+    rerender(<Select value={null} onChange={onChange} options={options} />);
+
+    expect(container.querySelector(".Select__single-value")).not.toBeInTheDocument();
+  });
+
+  it("should unselect value if no value is passed", async () => {
```
Comment (jim-docker) [inline src/renderer/components/select/select.test.tsx]: how is this test different from what's done on lines 109-111?

File: src/renderer/components/select/select.test.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    ];
+
+    const onChange = jest.fn();
+
+    const { container, rerender } = render(<Select value={options[0].value} onChange={onChange} options={options} />);
+    const selectedValueContainer = container.querySelector(".Select__single-value");
+
+    expect(selectedValueContainer.textContent).toBe(options[0].label);
+
+    rerender(<Select value={null} onChange={onChange} options={options} />);
```
Comment (jim-docker) [inline src/renderer/components/select/select.test.tsx]: value of `null` here means nothing is selected? What was happening when `selectedOption` was `undefined`?

**Review by jim-docker — DISMISSED** [review]

File: src/renderer/components/select/select.test.tsx (hunk 2 above)
Comment (DmitriyNoa) [reply src/renderer/components/select/select.test.tsx]: Yes. If an undefined was passed the old selected value was still selected, at least the label wasn't cleared.

Comment (Nokel81) [reply src/renderer/components/select/select.test.tsx]: Maybe it should, since both values are not in the list of options.

Comment (DmitriyNoa) [reply src/renderer/components/select/select.test.tsx]: Then it means there is no way to clear the selection programmatically?

Comment (Nokel81) [reply src/renderer/components/select/select.test.tsx]: How so? I am just proposing to add a new test case to cover `value={undefined}`. Or do you think that shouldn't also work to clear the value?

File: src/renderer/components/select/select.test.tsx (hunk 1 above)
Comment (DmitriyNoa) [reply src/renderer/components/select/select.test.tsx]: Cleaned that

File: src/renderer/components/select/select.test.tsx (hunk 2 above)
Comment (DmitriyNoa) [reply src/renderer/components/select/select.test.tsx]: Sorry, got it wrong. Added also test for undefined

**Review by Nokel81 — APPROVED** [review]

**Review by jim-docker — APPROVED** [review]

**Pull request merged**